phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Beans">
+    <description>The code standard for Beans.</description>
+    <file>.</file>
+    <exclude-pattern>dist/*</exclude-pattern>
+    <exclude-pattern>node_modules/*</exclude-pattern>
```

Comment:
May not be needed right now, but likely will be needed once other Node packages are added.

Reply:
Yes, there will be other node modules as we bring in gulp, linting, babel, etc.